BUILD.md

```diff
@@ -1,105 +1,131 @@
 WDT opensource build uses CMake 3.2.x or later
 
 This should build in a variety of environments (if it doesn't please open
-and issue or even better, contribute a patch)
+an issue or even better, contribute a patch)
 
 See also README.md Dependencies section
 (Inside facebook see https://our.intern.facebook.com/intern/wiki/WDT/OpenSource)
 
-Notes:
- On Ubuntu 14.04 - to get g++ 4.9
+# Notes:
+On Ubuntu 14.04 - to get g++ 4.9
+ ```
  sudo add-apt-repository ppa:ubuntu-toolchain-r/test
  sudo apt-get upgrade
+ ```
 If using a vmware image/starting fresh
+```
  sudo vmware-config-tools.pl  # to setup shared folders
-
-# Cmake 3.2 or later
-
-See below for mac (Cmake 3.3/head on a mac is recommended for Xcode support)
-
+```
+# Build Instructions
+__Install Cmake 3.2 or greater.__
+*See directions below for Mac (Cmake 3.3/head on a Mac is recommended for Xcode support)*
+```
 wget http://www.cmake.org/files/v3.2/cmake-3.2.3.tar.gz
 tar xvfz cmake-3.2.3.tar.gz
 cd cmake-3.2.3
 ./bootstrap --prefix=/usr --parallel=16 && make -j && sudo make install
-
-# Get folly
+```
+__Install folly.__
+```
 mkdir deps/; cd deps
 git clone https://github.com/facebook/folly.git
-
-# Getting pre built dependencies: glog-dev (which adds gflags, libunwind)
-# boost system, double conversion
-sudo apt-get install libgoogle-glog-dev libboost-system-dev \
-     libdouble-conversion-dev # and optionally: libjemalloc-dev
-
-# OR
-
-# If double-conversion isn't available:
-
+```
+__Install glog-dev (includes gflags, libunwind), boost system, double conversion__
+*libjemalloc-dev dependency is optional*
+```
+sudo apt-get install libgoogle-glog-dev libboost-system-dev libdouble-conversion-dev libjemalloc-dev
+```
+*If double-conversion isn't available via apt-get:*
+```
 git clone https://github.com/floitsch/double-conversion.git
 cd double-conversion; cmake . ; make -j && sudo make install
+```
 
+__Build gflags and glog from source__
 
-# Get and install gflags and glog - it's important to build and configure
-# gflags correctly for glog to pick it up and avoid linking errors later
-# or getting a wdt without flags working
+*It's important to build and configure gflags correctly for glog to pick it up and avoid linking errors later or getting a wdt without flags working*
```

Comment:
leave the file with max 80 columns so it's still readable with more/less/vi command line